### ISSUE TITLE
Show reason why group restriction not possible for that app

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#4864]
Papercut from https://github.com/nextcloud/server/issues/2672

cc @eppfel @MariusBluem because you always asked for this

**Comment by wiswedel** [comment]
@MorrisJobke Is this still of interest or can it be closed? Can't there be several reasons - none of them giving the reader any practical benefit?

**Comment by MorrisJobke** [comment]
> Can't there be several reasons - none of them giving the reader any practical benefit?

Yes there can be different and it would be helpful to communicate why given features are not there IMO.

**Comment by szaimen** [comment]
I am going to close this since there doesn't seem to be a lot of interest (no upvotes) and no respond since around 2 years. Please reopen if you still want to implement this.

**Comment by joshtrichards** [comment]
I suppose these are the reasons:
https://github.com/nextcloud/server/blob/7ddad46137c17e3c272adbcf02ca29d7e8b2d6bd/apps/settings/src/mixins/AppManagement.js#L74-L82

https://github.com/nextcloud/server/blob/7ddad46137c17e3c272adbcf02ca29d7e8b2d6bd/apps/settings/src/components/AppStoreSidebar/AppDetailsTab.vue#L15

https://github.com/nextcloud/server/blob/7ddad46137c17e3c272adbcf02ca29d7e8b2d6bd/lib/private/App/AppManager.php#L35-L44

Apps this applies to:

https://github.com/search?q=org%3Anextcloud+path%3A%2F%28%5E%7C%5C%2F%29info%5C.xml%24%2F+filesystem+OR+prelogin+OR+authentication+OR+logging+OR+prevent_group_restrictions&type=code

For many (most?) of these it's somewhat obvious why they can't be limited. But I can see how a few might be surprising (e.g. Collectives, Photos, Maps, Community Document Server, Office Online, Activity).
